pkgs/development/python-modules/scikit-fuzzy/default.nix

```
@@ -2,51 +2,26 @@
   lib,
   buildPythonPackage,
   fetchFromGitHub,
-  fetchpatch,
-  pythonAtLeast,
   setuptools,
   matplotlib,
   networkx,
-  nose,
   numpy,
   scipy,
   pytest7CheckHook,
 }:
 
-buildPythonPackage rec {
+buildPythonPackage {
   pname = "scikit-fuzzy";
-  version = "unstable-2022-11-07";
+  version = "unstable-2023-09-14";
```

When updating please use the new versioning schema for unstable version.

```suggestion
  version = "0.4.2-unstable-2023-09-14";
```